frontend/src/metabase/lib/query/field.js

```diff
@@ -3,6 +3,7 @@ import { mbqlEq } from "./util";
 
 import type { Field as FieldReference } from "metabase/meta/types/Query";
 import type { Field, FieldId, FieldValues } from "metabase/meta/types/Field";
+import type { Value } from "metabase/meta/types/Dataset";
```

Comment:
yep, the first parameter of `getHumanReadableValue` uses it and dumb Flow doesn't complain about missing type imports